source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5708,9 +5709,23 @@ class OverflowAction extends Action {
                 contextMenu.items.add(menuItem);
             }
 
+            contextMenu.onClose = () => {
+                if (this.renderedElement) {
+                    this.renderedElement.ariaExpanded = "false";
+                }
```

Comment:
alternatively could write as 

```suggestion
                this.renderedElement?.ariaExpanded = "false";
```

Reply:
This didn't work originally because you can't do property assignment on an optional variable. I just updated to use `setAttribute` here as well :)